src/serializable.rs

```diff
@@ -39,6 +39,7 @@ impl<T: Serialize, const N: usize, const M: usize> Serializable<T, N> for [[T; N
     }
 }
 
+#[cfg(feature = "std")]
```

Comment:
This can be enabled even in no-std environment using `alloc::vec::Vec`. Maybe introduce `alloc` feature in addition that enables `serde`'s `alloc` feature in addition to `std`?

Reply:
sadly looks like this project is unmaintained

Reply:
It does, but the author is still active on GitHub periodically, so :crossed_fingers: